wandb/env.py

```diff
@@ -409,3 +410,12 @@ def disable_git(env: Optional[Env] = None) -> bool:
     if isinstance(val, str):
         val = False if val.lower() == "false" else True
     return val
+
+
+def disable_job_creation(env: Optional[Env] = None) -> bool:
```

Comment:
you could completely remove this function and just do os.environ.get(...) in settings up to you

Reply:
Oh, wow, nice!